docs/reference/ilm/ilm-index-lifecycle.asciidoc

```diff
@@ -6,13 +6,16 @@
 <titleabbrev>Index lifecycle</titleabbrev>
 ++++
 
-{ilm-init} defines four index lifecycle _phases_:
+{ilm-init} defines five index lifecycle _phases_:
 
 * **Hot**: The index is actively being updated and queried.
 * **Warm**: The index is no longer being updated but is still being queried.
 * **Cold**: The index is no longer being updated and is seldom queried. The
 information still needs to be searchable, but it's okay if those queries are
 slower.
+* **Frozen**: The index is no longer being updated and is even more seldom queried. The
```

Comment:
Hmm...`more seldom` is an odd phrasing, in the way that a double negative is. How about rewriting `seldom queried` in `Cold` to `queried infrequently` and in `Frozen` to `queried rarely`?